package.json

```diff
@@ -77,7 +77,7 @@
     "@babel/runtime": "7.5.4",
     "@octokit/rest": "16.28.4",
     "absolutify": "0.1.0",
-    "amp-by-example": "0.2.13",
+    "amp-by-example": "^0.2.15",
```

Comment:
Is this a non-fixed version intenionally?

Reply:
nope. thanks for spotting. updated the version as well.